web/web.go

```diff
@@ -52,6 +53,7 @@ import (
 	"github.com/prometheus/tsdb"
 	"golang.org/x/net/netutil"
 
+	"github.com/prometheus/client_model/go"
```

Comment:
Nit: This should be in the previous block. (We only put packages from the same Git repo in the last block.)